src/__tests__/getTemplateLocation.test.ts

```diff
@@ -1,10 +1,17 @@
-import { DEFAULT_TEMPLATE_PATH, getTemplateLocation } from "../utils";
+import {
+  DEFAULT_TEMPLATE_PATH,
+  getTemplateLocation
+} from "../utils/getTemplateLocation";
 
-afterEach(() => {
-  jest.resetModules();
-});
+describe("my thing", () => {
```

Comment:
should probably use something more descriptive 😂

Reply:
🤦‍♂️ that's embarrassing